localrec/__init__.py

```diff
@@ -30,34 +30,33 @@
 """
 import os
 import pyworkflow.em
-from bibtex import _bibtex # Load bibtex dict with references
-from opic.constants import *
-from opic.convert import *
+from pyworkflow.utils import Environ
 
+from bibtex import _bibtex # Load bibtex dict with references
```

Comment:
This is no longer needed. As far as it is named "bibtex.py" Scipion will grab the references from there. ;)